pkg/descheduler/descheduler.go

```diff
@@ -323,6 +324,9 @@ func RunDeschedulerStrategies(ctx context.Context, rs *options.DeschedulerServer
 			eventRecorder,
 		)
 
+		var parentCloser func()
+		ctx, _, parentCloser = tracing.StartSpan(ctx, "descheduler_parent", "deschedule")
+		defer parentCloser()
```

Comment:
this context already has a `cancel()` function that is already deferred, so I don't think `parentCloser()` is necessary

Reply:
also (commented below re:this), but we can just use 1 named tracer, so `descheduler_parent` could just be `descheduler`. It might be better to actually name our tracer something namespaced like `sigs.k8s.io/descheduler` but I'll double check that

Reply:
@harshanarayana does this sound good to you?